docs/integrations/veeam/README.md

```diff
@@ -5,10 +5,10 @@ When using Veeam Backup and Replication, you can use S3 compatible object storag
 ## Prerequisites
```

Comment:
This page was migrated to minio/docs/source/integrations/using-minio-with-veeam.md - needs fixes there too.

Reply:
Will handle in a separate PR. Adding an issue to that repo.

Reply:
https://github.com/minio/docs/issues/581